test/test-coordinator.js

```diff
@@ -156,9 +156,15 @@ TestCoordinator.prototype.start = function start(callback) {
     var self = this;
     callback = callback || function(){};
 
-    self.once('joined', function joined(){
+    self.on('joined', function joined(){
         // Set firstJoinDate to current Date if not already set.
-        self.firstJoinDate = self.firstJoinDate || new Date();
+        self.firstJoinTime = self.firstJoinTime || Date.now();
+        self.lastJoinTime = Date.now();
+        console.log('got joined: ', self.firstJoinTime, self.lastJoinTime);
```

Comment:
I don't think we want to start logging from tests. If there is a good reason to always log this we can keep it, but I'd rather see that we put all data needed to debug a failure in the failure of a test.